README.md

```diff
@@ -34,9 +34,19 @@ cd workshop
 ./geopython-workshop-ctl.sh stop
 ```
 
-Windows (Powershell):
+Windows (Command Prompt):
```

Comment:
Change to "Windows (Command Prompt or PowerShell") - the script works  in PowerShell (I'll need to check Cmd)

Reply:
ah indeed, `docker compose version >NUL` this doesn't work on cmd

Reply:
Confimed working on cmd and powershell